otter/log/formatters.py

```diff
@@ -237,6 +239,22 @@ def audit_log_formatter(eventDict, timestamp, hostname):
     return audit_log_params
 
 
+@singledispatch
+def serialize_to_jsonable(obj):
+    """
+    Serialize any object to a JSONable form
+    """
+    return None
+
+
+@serialize_to_jsonable.register(basestring)
+def serialize_str(s):
+    """
+    Serialize string
+    """
+    return s
```

Comment:
oh, and we should do this for all of the pyrsistent data structures, but not necessary in this branch if you don't want to :)